apps/ledger-live-desktop/src/renderer/screens/settings/sections/Developer/MockAppUpdate.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useCallback, useState } from "react";
+import { getEnv, setEnv } from "@ledgerhq/live-env";
+import Track from "~/renderer/analytics/Track";
+import Switch from "~/renderer/components/Switch";
+
+const MockAppUpdate = () => {
+  const [enabled, setEnabled] = useState(getEnv("MOCK_APP_UPDATE"));
```

Comment:
you can use `useEnv` and then a `setEnv` will automatically will have the state reflecting this for you.

https://github.com/LedgerHQ/ledger-live/blob/develop/libs/ledger-live-common/src/hooks/useEnv.ts

Reply:
thanks will change